### PR TITLE
Week1 prep for spotlist frontend

### PR DESCRIPTION
**Pull request by mchristia**
Focused on frontend for basic function and view:

- [x] homepage with a button for search for spot
- [x] search page shows list of all spots (currently hard coded, no real design)
- [x] search page show a filter option with no function yet

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,21 +1,78 @@
...
+            "break" : "Beach and Reef",
+            "level" : "All",
+
+        }]
+
   return (
       <Switch>
-        <div>
-            <Route path = "/home">
+            <Route path = "/" exact>
```
Comment (christian-porsch) [inline frontend/src/App.js:70]: I would keep home and delete the other one

File: frontend/src/component/SpotItem.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components/macro";
+
+export default function SpotItem({spot}){
```
Comment (christian-porsch) [inline frontend/src/component/SpotItem.js:3]: you should also name the props spots like you did in your app.js props

**Review by christian-porsch — COMMENTED** [review]
good work!

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,21 +1,78 @@
...
+            "geography" : {
+                "longitude" : 1,
+                "latitude" : 1,
+                "continent" : "Europa",
+                "country" : "France",
+                "Region" : "Basque",
+            },
+            "break" : "Beach",
+            "level" : "All",
+
```
Comment (fabianschmauder) [inline frontend/src/App.js:20]: maybe you should add and use prettier ;)

**Review by fabianschmauder — COMMENTED** [review]
use and add prettier ;)

**Pull request merged**